### ISSUE TITLE
cant get specto installed ?

### ISSUE THREAD

**Issue by JackDinn** [mrknow/specto#37]
it just says there is no connection when i add the source http://kodi.filmkodi.com

**Comment by BankruptEd** [comment]
I don't know exactly what that means, but I do suspect that Specto is no
longer being maintained. Maybe this is the problem?

**Comment by JackDinn** [comment]
I found it on the Merlin repo atm, looks like http://filmkodi.com/ is dead.

**Comment by BankruptEd** [comment]
Yup, that could well be the case. Specto and a bunch of other add-ons have
ceased to be recently. We will just have to sit tight and see where it
ends. You might want to try Terrarium TV in the meantime. It's not a Kodi
add-on, but just a stand alone app you can install on your android box (if
u have one) or phone or laptop. Covers you for television and movies quite
nicely.

**Comment by JackDinn** [comment]
yea iv got TTV, lovely sources, lots of 1080p's etc & no need for pairing, only prob is a lot of them have buffering problems but im on the reddit sub asking questions.

**Comment by BankruptEd** [comment]
Really? I have very few buffering issues (one of the reasons I love it) but
I did notice it doesn't work as well on my android box with android build
4.4.2. It works much better on my more recent box with android 6.0. Anyway,
hopefully you get it all straightened out. Peace.

On Jul 24, 2017 11:39 AM, "JackDinn" <notifications@github.com> wrote:

yea iv got TTV, lovely sources, lots of 1080p's etc & no need for pairing,
only prob is a lot of them have buffering problems but im on the reddit sub
asking questions.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/mrknow/specto/issues/37#issuecomment-317463339>, or mute
the thread

**Comment by juanjotoronja** [comment]
Elysium is an identical replacement, it has "My Elysium" section to add your favorite TV shows and movies.

**Comment by BankruptEd** [comment]
Two words "Terrarium TV". Is all you need my friends.
